neo/rawio/openephysbinaryrawio.py

```diff
@@ -481,105 +481,108 @@ def explore_folder(dirname, experiment_names=None):
+                        stream_name = node_name + '#' + oe_stream_name
 
-                    # Updates for OpenEphys v0.6:
-                    # In new vesion (>=0.6) timestamps.npy is now called sample_numbers.npy
-                    # see https://open-ephys.github.io/gui-docs/User-Manual/Recording-data/Binary-format.html#continuous
-                    sample_numbers = recording_folder / 'continuous' / info['folder_name'] / \
-                        'sample_numbers.npy'
-                    if sample_numbers.is_file():
-                        timestamp_file = sample_numbers
-                    else:
-                        timestamp_file = recording_folder / 'continuous' / info['folder_name'] / \
-                            'timestamps.npy'
-                    timestamps = np.load(str(timestamp_file), mmap_mode='r')
-                    timestamp0 = timestamps[0]
-                    t_start = timestamp0 / info['sample_rate']
-
-                    # TODO for later : gap checking
-                    signal_stream = info.copy()
-                    signal_stream['raw_filename'] = str(raw_filename)
-                    signal_stream['dtype'] = 'int16'
-                    signal_stream['timestamp0'] = timestamp0
-                    signal_stream['t_start'] = t_start
-
-                    recording['streams']['continuous'][stream_name] = signal_stream
-
-            if (root / 'events').exists() and len(rec_structure['events']) > 0:
-                recording['streams']['events'] = {}
-                for info in rec_structure['events']:
-                    oe_stream_name = Path(info["folder_name"]).name # remove trailing slash
-                    stream_name = node_name + '#' + oe_stream_name
-
-                    event_stream = info.copy()
-                    for name in _possible_event_stream_names:
-                        npy_filename = root / 'events' / info['folder_name'] / f'{name}.npy'
-                        if npy_filename.is_file():
-                            event_stream[f'{name}_npy'] = str(npy_filename)
+                        event_stream = info.copy()
+                        for name in _possible_event_stream_names:
+                            npy_filename = root / 'events' / info['folder_name'] / f'{name}.npy'
+                            if npy_filename.is_file():
+                                event_stream[f'{name}_npy'] = str(npy_filename)
 
-                    recording['streams']['events'][stream_name] = event_stream
+                        recording['streams']['events'][stream_name] = event_stream
 
-            folder_structure[node_name]['experiments'][experiment_id]['recordings'][recording_id] \
-                = recording
+                folder_structure[node_name]['experiments'][experiment_id]['recordings'][recording_id] \
+                    = recording
+    except Exception as e:
+        raise Exception(f"Error while exploring Open Ephys folder {dirname}. Invalid folder structure?")
```

Comment:
```suggestion
        raise Exception(f"Error {e} while exploring Open Ephys folder {dirname}. Invalid folder structure.")
```

Do we want to actually share the e for the user so they can see what exactly triggered the exception or do you think that is too much info? Also you mean `.` or `?` at the end.

Reply:
True. The issue tracker often involves these folder issues. :)